### PR TITLE
Telemetry dashboard: add error rate, update every 3h

### PR DESCRIPTION
**Pull request by dcposch**
screen shot 2016-08-21 at 2 07 43 am

### CODE REVIEW

**Comment by dcposch** [comment]
@feross

File: server/views/telemetry-dashboard.pug [hunk 1]
```diff
@@ -10,25 +10,38 @@ block body
...
+      svg#chart-retention
+
+    h2 Quality
+    section
+      h3 Error rate
+      p The fraction of telemetry reports that have uncaught errors. The app sends a report every time it starts, plus once every 24 hours.
+      svg#chart-error
+
+    //-
+      h3 Most common errors this week
```
Comment (dcposch) [inline server/views/telemetry-dashboard.pug]: Coming Soon™

File: server/desktop-api.js [hunk 2]
```diff
@@ -31,15 +31,17 @@ try { mkdirp.sync(TELEMETRY_PATH) } catch (err) {}
...
  * the data in summary.json.
  *
- * This is a poor-man's cron-job :) It's nice because it keeps everything in
- * this repo, though.
+ * This is a poor-man's cron-job. Not running in development by default to
+ * avoid hitting Github's API too often.
  */
-summarizeTelemetry()
-setInterval(summarizeTelemetry, 24 * 3600 * 1000)
+if (process.env.NODE_ENV === 'production') {
```
Comment (feross) [inline server/desktop-api.js]: Nit-pick: `config.isProd`

**Comment by feross** [comment]
LGTM!!

**Comment by feross** [comment]
This caused this bug: screen shot 2016-08-21 at 4 45 47 pm

**Comment by feross** [comment]
https://github.com/feross/webtorrent-www/commit/56ba491f8a714037543d87230013ebac6369f47f

https://github.com/feross/webtorrent-www/commit/bfb8f48a17e2b5245cd4907e0a9ae9a27bde9868

**Pull request merged**